.github/workflows/build-docker-corsika-simtelarray-image.yml

```diff
@@ -60,17 +100,107 @@ jobs:
             type=semver,pattern={{major}}.{{minor}}.{{patch}}
             type=schedule,pattern={{date 'YYYYMMDD'}}
             type=raw,value={{date 'YYYYMMDD-HHmmss'}}
-          images: ${{ env.REGISTRY }}/gammasim/simtools-corsika-sim-telarray-${{ matrix.hadronic_model }}-${{ matrix.production }}-${{ matrix.version }}
+          images: ${{ env.REGISTRY }}/gammasim/simtools-corsika-sim-telarray-${{ env.LABELS }}
           flavor: latest=true
 
-      - name: Build and push Docker image
+      - name: Build and push image
         uses: docker/build-push-action@v6
         with:
           context: .
           platforms: linux/amd64,linux/arm64/v8
-          build-args: '{"BUILD_OPT": ${{ matrix.production }}, "HADRONIC_MODEL": "qgs2"}'
+          build-args: |
+           BUILD_OPT=${{ matrix.production.build_opt }}
+           EXTRA_DEF=${{ matrix.production.extra_def }}
+           HADRONIC_MODEL=${{ matrix.hadronic_model }}
+           SIMTEL_VERSION=${{ matrix.version.simtel_version }}
+           CORSIKA_VERSION=${{ matrix.version.corsika_version }}
+           BERNLOHR_VERSION=${{ matrix.version.bernlohr_version }}
           push:
             ${{ github.event_name == 'release' || github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch' }}
           file: ./docker/Dockerfile-simtelarray
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}-corsika-sim-telarray-${{ matrix.hadronic_model }}-${{ matrix.production }}-${{ matrix.version }}
+          labels: ${{ steps.meta.outputs.labels }}-corsika-sim-telarray-${{ env.LABELS }}
+
+  build-production-images-optimized:
+    runs-on: ubuntu-latest
+    needs: download-corsika
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        version:
+          - { simtel_version: '240205', corsika_version: '77500', bernlohr_version: '1.67' }
+#          - { simtel_version: '240927', corsika_version: '77550', bernlohr_version: '1.68' }
```

Comment:
I assume you use the older one here and the newer one above just to test both and it's not a mistake, right?

Reply:
I have commented out those options which we are not using to keep the number of github workflows running at a necessary minimum. I however would like to keep the options in the workflow file to indicate how to expand the build matrix. 

We have used until now always 77500 with 1.67 (and not the newer package).

Reply:
Right, but above you commented out 1.67 and left 1.68. That's why I asked.

Reply:
Yes! This is from testing. I've changes this now and all are at 1.67.

Reply:
OK!